doc/tools/nebula-algorithm/na-ug-limitations.md

```diff
@@ -0,0 +1,3 @@
+# 使用限制
```

Comment:
how about put this content directly at the bottom of  `na0ug-what-si-nebula-algorithm.md`
A seperate `na-ug-limitations.md` looks unnecessary - and in most cases  will be ignored.